### ISSUE TITLE
Call to activation_visualization seems to be running forever with no result

### ISSUE THREAD

**Issue by gioCanelita** [InFoCusp/tf_cnnvis#72]
Hi

I am relatively new to Tensorflow and have been playing with the Cifar-10 model from tensorflow. I am now trying to use the activation_visualization function within the cifar10_eval.py , have tried different parameters in the function, but it doesnt seem to work for me. When I run the code, and gets to the point of calling the function, the program just sits there, seems to be running but not doing anything. It is probably a silly mistake that i can't just see :/, hopefully you can help me? I would really appreciate your help.
At the moment, I am tryng to use it here:

    def eval_once(saver, summary_writer, top_k_op, summary_op, logits, labels, images, top_k_predict_op):

      images_placeholder = tf.placeholder(tf.float32, shape=(1,
                                                         256, 256, 3))
      with tf.Session() as sess:     
      ckpt = tf.train.get_checkpoint_state(FLAGS.checkpoint_dir)
      print(ckpt)

      if ckpt and ckpt.model_checkpoint_path:
        # Restores from checkpoint
        print(ckpt.model_checkpoint_path)
        saver.restore(sess, ckpt.model_checkpoint_path)
        # Assuming model_checkpoint_path looks something like:
        #   /my-favorite-path/cifar10_train/model.ckpt-0,
        # extract global_step from it.
        global_step = ckpt.model_checkpoint_path.split('/')[-1].split('-')[-1]
      else:
        print('No checkpoint file found')
        return
  
      layers = ['c']
      #feed_dict = {images_placeholder:images.eval()}
      print('creating activation image')
      is_success = activation_visualization(sess_graph_path = tf.get_default_graph(), value_feed_dict = images.eval(),
                                          layers=layers, path_logdir=os.path.join(FLAGS.eval_dir,"Log/ActivationImages"), 
                                          path_outdir=os.path.join(FLAGS.eval_dir,"Output"))

Thanks again!

**Comment by gioCanelita** [comment]
I believe the problem is the images.eval(), as this code uses batches, and for some reason that makes the eval to hang.
thanks!
